js_modules/dagster-ui/packages/ui-core/src/runs/RunUtils.tsx

```diff
@@ -115,6 +119,73 @@ export async function handleLaunchResult(
   }
 }
 
+export async function handleLaunchMultipleResult(
+  result: void | null | LaunchMultipleRunsMutation['launchMultipleRuns'],
+  history: History<unknown>,
+  options: {behavior: LaunchBehavior; preserveQuerystring?: boolean},
+) {
+  if (!result) {
+    showCustomAlert({body: `No data was returned. Did dagster-webserver crash?`});
+    return;
+  }
+  const successfulRunIds: string[] = [];
+
+  // show corresponding toasts
+  if (result.__typename === 'LaunchMultipleRunsSuccess') {
+    for (const individualResult of result.runs) {
```

Comment:
We should probably collapse multiple run launches into a single toast, we can link to the runs page filtered to that set of runs.

Reply:
Oh we're already navigating to that page. I think a single toast makes more sense since they can already see all of the runs on that page, no need to toast for each run.

Reply:
https://github.com/dagster-io/dagster/pull/25633/files#diff-ab6a18c9af0c0cd283c056df4b1dca2a42412fac74a4c0198e95bd685affbd0bR192-R199